QsCompiler.sln

```diff
@@ -270,6 +296,7 @@ Global
 		{D50583DF-FBEF-45EF-B523-70B2CDBE1DD1} = {76BA96DA-DC1E-4315-A3ED-5F0700A79812}
 		{2E331781-F7ED-4EF1-8451-896636C6D93A} = {76BA96DA-DC1E-4315-A3ED-5F0700A79812}
 		{D2E36476-A65F-4310-9C4C-B721BCC47B00} = {6077A717-50BF-4F87-B439-CA549AF6A4AE}
+		{A83A9BA2-A774-4F07-B801-5A4A73598F62} = {D2E36476-A65F-4310-9C4C-B721BCC47B00}
```

Comment:
This line is indicating that there should be another project besides Library1.csproj under src\QsCompiler\TestTargets\Libraries, but no such project is listed there, earlier in this solution file. If there is supposed to be a project, you could add it to the solution and to the PR, otherwise I think all references to {A83A9BA2-A774-4F07-B801-5A4A73598F62} have to be removed for it to build correctly.

Reply:
Duly removed. Odd that this is ignored when I build locally.

Reply:
It build locally for me as well, which is strange. Maybe VSO has a more restrictive process.